vumi_http_proxy/test/test_servicemaker.py

```diff
@@ -0,0 +1,24 @@
+from vumi_http_proxy.servicemaker import Options, ProxyWorkerServiceMaker
+from twisted.trial import unittest
+
+
+class TestOptions(unittest.TestCase):
+    def test_defaults(self):
+        options = Options()
+        options.parseOptions([])
+        self.assertEqual(options["port"], 8080)
+        self.assertEqual(str(options["interface"]), "0.0.0.0")
+
+    def test_override(self):
+        options = Options()
+        options.parseOptions(["--port", 8000])
+        options.parseOptions(["--interface", "127.0.0.1"])
+        self.assertEqual(options["port"], "8000")
+        self.assertEqual(str(options["interface"]), "127.0.0.1")
+
+
+class TestProxyWorkerServiceMaker(unittest.TestCase):
+    def test_makeService(self):
+        self.assertEqual(
+          str(ProxyWorkerServiceMaker.makeService),
+          '<unbound method ProxyWorkerServiceMaker.makeService>')
```

Comment:
We should actually call `makeService` (with some options) and then assert things about the service we get back, e.g.:
```python
    options = Options()
    options.parseOptions([])
    servicemaker = ProxyWorkerServiceMaker()
    service = servicemaker.makeService(options)
    self.assertEqual(service.factory, vumi_http_proxy.http_proxy.ProxyFactory)
    self.assertEqual(service.endpoint._interface, '0.0.0.0')
    self.assertEqual(service.endpoint._port, 8080)
```